samples/create-buffers-around-points/src/main/java/com/esri/arcgismaps/sample/createbuffersaroundpoints/screens/CreateBuffersAroundPointsScreen.kt

```diff
@@ -0,0 +1,190 @@
+                        style = MaterialTheme.typography.bodyLarge,
+                    )
+                }
+                // MapView: display the map and handle taps
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.arcGISMap,
+                    graphicsOverlays = mapViewModel.graphicsOverlays,
+                    onSingleTapConfirmed = { tapEvent ->
+                        tapEvent.mapPoint?.let { mapPoint: Point ->
+                            mapViewModel.onMapTapped(mapPoint)
+                        }
+                    }
+                )
+
+
+                // Controls row: union switch and clear button
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(12.dp),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Row(
+                        horizontalArrangement = Arrangement.spacedBy(8.dp),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Text(text = "Union", style = MaterialTheme.typography.bodyLarge)
+                        Switch(
+                            checked = shouldUnion,
+                            onCheckedChange = { checked -> mapViewModel.updateUnion(checked) }
+                        )
+                    }
+
+                    Row {
+                        OutlinedButton(onClick = { mapViewModel.clearAll() }) {
+                            Icon(Icons.Filled.Clear, contentDescription = "Clear buffers")
+                            Text(text = " Clear", modifier = Modifier.padding(start = 8.dp))
+                        }
+                    }
+                }
+
+                // Message dialog from the view model
+                mapViewModel.messageDialogVM.apply {
+                    if (dialogStatus) {
+                        MessageDialog(
+                            title = messageTitle,
+                            description = messageDescription,
+                            onDismissRequest = ::dismissDialog
+                        )
+                    }
+                }
+            }
+
+            // Buffer radius input dialog
+            if (isInputVisible) {
+                AlertDialog(
+                    onDismissRequest = { },
+                    title = { Text(text = "Buffer radius (miles)") },
+                    text = {
+                        Column {
+                            Text(text = "Enter a radius between 0 and 300 miles.")
+                            TextField(
+                                value = radiusInput,
+                                onValueChange = { radiusInput = it },
+                                label = { Text("Radius in miles") }
+                            )
```

Comment:
Optional accessibility suggestion: Configure the text field to use a num pad: 
```suggestion
                            TextField(
                                value = radiusInput,
                                onValueChange = { radiusInput = it },
                                label = { Text("Radius in miles") },
                                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number)
                            )
```